.github/workflows/gobra.yml

```diff
@@ -415,7 +415,7 @@ jobs:
           includePaths: ${{ env.includePaths }}
           assumeInjectivityOnInhale: ${{ env.assumeInjectivityOnInhale }}
           checkConsistency: ${{ env.checkConsistency }}
-          chop: 10
+          # chop: 10
```

Comment:
```suggestion
          # Due to a bug introduced in https://github.com/viperproject/gobra/pull/776,
          # we must currently disable the chopper, otherwise we well-founded orders
          # for termination checking are not available at the chopped Viper parts.
          # chop: 10
```